{"uuid": "41021ff5-abf9-4083-bacc-6a1513c9571f", "children": ["0fdb43ae-86b4-4e3a-82b8-5e3fde5642a9", "f920e5cb-9de9-4a2e-9f2b-e2b8a9750f1b"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: failed to change window state to 'normal', current state is 'maximized'\n  (Session info: chrome=106.0.5249.103)\nStacktrace:\nBacktrace:\n\tOrdinal0 [0x00C4DF13+2219795]\n\tOrdinal0 [0x00BE2841+1779777]\n\tOrdinal0 [0x00AF423D+803389]\n\tOrdinal0 [0x00AE0021+720929]\n\tOrdinal0 [0x00ADEE74+716404]\n\tOrdinal0 [0x00ADDCF9+711929]\n\tOrdinal0 [0x00B5FBAF+1244079]\n\tOrdinal0 [0x00B3E7FC+1107964]\n\tOrdinal0 [0x00B4F192+1175954]\n\tOrdinal0 [0x00B3E616+1107478]\n\tOrdinal0 [0x00B17F89+950153]\n\tOrdinal0 [0x00B18F56+954198]\n\tGetHandleVerifier [0x00F42CB2+3040210]\n\tGetHandleVerifier [0x00F32BB4+2974420]\n\tGetHandleVerifier [0x00CE6A0A+565546]\n\tGetHandleVerifier [0x00CE5680+560544]\n\tOrdinal0 [0x00BE9A5C+1808988]\n\tOrdinal0 [0x00BEE3A8+1827752]\n\tOrdinal0 [0x00BEE495+1827989]\n\tOrdinal0 [0x00BF80A4+1867940]\n\tBaseThreadInitThunk [0x7650FA29+25]\n\tRtlGetAppContainerNamedObjectPath [0x772F7A9E+286]\n\tRtlGetAppContainerNamedObjectPath [0x772F7A6E+238]\n\n", "trace": "  File \"C:\\Users\\Sergey\\diplom_wolk\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Sergey\\diplom_wolk\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1121, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Sergey\\diplom_wolk\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 900, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\Sergey\\diplom\\conftest.py\", line 11, in driver\n    return init_driver()\n  File \"C:\\Users\\Sergey\\diplom\\conftest.py\", line 18, in init_driver\n    driver.maximize_window()\n  File \"C:\\Users\\Sergey\\diplom_wolk\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 596, in maximize_window\n    self.execute(command, None)\n  File \"C:\\Users\\Sergey\\diplom_wolk\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 428, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Sergey\\diplom_wolk\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 243, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1665592607344, "stop": 1665592613783}], "start": 1665592607344, "stop": 1665592616263}